station/utils.py

```diff
@@ -90,6 +91,16 @@ def get_planner_command(directory=None):
 def get_receiver_command(directory=None):
     return _get_command(directory, "receiver.py")
 
+def get_updater_command(directory=None):
```

Comment:
Maybe you should refactor ```_get_command``` function. Now, this function has the same content only without last line.

Reply:
Nah, it was subtly different. It didn't have the filename parameter. But I get your point. I've refactored the code. There's just one function `_get_command()` and its code is not duplicated. Is it any better now?